vms/platformvm/vm_test.go

```diff
@@ -2230,7 +2258,10 @@ func TestBaseTx(t *testing.T) {
 	require.Equal(vm.TxFee, totalInputAmt-totalOutputAmt)
 	require.Equal(sendAmt, key1OutputAmt)
 
-	require.NoError(vm.Network.IssueTx(context.Background(), baseTx))
+	vm.ctx.Lock.Unlock()
+	require.NoError(vm.issueTx(context.Background(), baseTx))
+	require.NoError(vm.issueTx(context.Background(), baseTx))
```

Comment:
Was this intended as a test that `ErrDuplicateTx` is checked for? or was this unintentional

Reply:
Unintentional, I unstaged this diff